lib/rules/color-function-notation/index.js

```diff
@@ -50,6 +52,10 @@ const rule = (primary, secondaryOptions, context) => {
 		const ignoreWithVarInside = optionsMatches(secondaryOptions, 'ignore', 'with-var-inside');
 
 		root.walkDecls((decl) => {
+			if (!LEGACY_NOTATION_FUNCTION.test(decl.value)) return;
+
+			if (primary === 'modern' && !COMMA.test(decl.value)) return;
```

Comment:
[suggest] Simply using `.includes()` seems faster and more readable than using a regex in this case. Any thoughts?

```suggestion
			if (primary === 'modern' && !decl.value.includes(',')) return;
```